Sources/TSCBasic/Process.swift

```diff
@@ -23,6 +23,30 @@ import Dispatch
 
 import _Concurrency
 
+public struct ProcessEnvironmentBlock {
```

Comment:
Not sure if `Block` brings much if anything valuable to the naming. Also, this should include a doc comment.
```suggestion
/// Windows is case insensitive for the environment control block. This caused issues when Swift is used with 
/// programs which are incorrectly cased (e.g. emacs).  This type Introduces an explicit wrapper type for 
/// Windows to make the lookup case insensitive, canonicalising the name to lowercase.  This allows us to treat 
/// `Path` and `PATH` identically (along with any other environment variable and case matching) which respects /// the Windows behaviour.
public struct ProcessEnvironment {
```

Reply:
Now looking at `ProcessEnv`, why not actually move this over there and call it `ProcessEnv.Storage`?

Reply:
That's fair, but IMO the doc comment suggestion still holds.